src/components/tabs/README.md

```diff
@@ -57,17 +57,17 @@ When `<b-tabs>` is in `card` mode, each `<b-tab>` sub-component will automatical
   <b-card no-body>
     <b-tabs card>
       <b-tab no-body title="Picture 1">
-        <b-card-img bottom src="https://picsum.photos/600/200/?image=21"></b-card-img>
+        <b-card-img bottom src="https://picsum.photos/600/200/?image=21" alt="img"></b-card-img>
```

Comment:
Maybe "Image 21"?

Reply:
Maybe... just as long as there is an `alt` attribute

Reply:
Should probably make sure all the card images in the card docs also have `alt` attributes

Reply:
Card docs updated